mne/utils/__init__.py

```diff
@@ -1,6 +1,6 @@
 # # # WARNING # # #
-# This list must also be updated in doc/_templates/class.rst if it is
-# changed here!
+# This list must also be updated in doc/_templates/autosummary/class.rst if it
+# is changed here!
```

Comment:
@larsoner I think the location of ```class.rst``` has changed?

Reply:
Indeed it has, thanks for the fix